intelligence/ts/src/engines/common/model.ts

```diff
@@ -22,13 +22,14 @@ const STALE_TIMEOUT_MS = 24 * 60 * 60 * 1000; // 24 hours.
 
 interface ModelResponse {
   is_supported: boolean;
-  engine_model: string | undefined;
-  model: string | undefined;
+  engine_model?: string;
+  model?: string;
+  vram?: number;
 }
 
-export const cacheStorage: CacheStorage = isNode ? new NodeCacheStorage() : new WebCacheStorage();
+const cacheStorage: CacheStorage = isNode ? new NodeCacheStorage() : new WebCacheStorage();
 
-async function updateModel(model: string, engine: string): Promise<Result<string>> {
+async function updateModel(model: string, engine: string): Promise<Result<ModelInfo>> {
```

Comment:
Should we rename `ModelInfo` to `ModelConfig` to align better with the name of the API endpoint?

Reply:
Probably best to do that in a separate PR